File: src/main/java/io/jsql/sql/handler/data_mannipulation/Mhandler.java
package io.jsql.sql.handler.data_mannipulation;

import io.jsql.sql.OConnection;

/**
 * Created by 长宏 on 2017/3/18 0018.
 * HANDLER tbl_name READ `PRIMARY` ...
 */
public class Mhandler {
    public static boolean isme(String sql) {
        String sqll = sql.toUpperCase().trim();
        String list[] = sqll.split("\\s+");
        return list.length > 0 && list[0].equals("HANDLER");
    }

    public static void handle(String sql, OConnection c) {
        c.writeok();
    }
}

File: src/main/java/io/jsql/sql/handler/data_mannipulation/Mhandler.kt
/*
 * Java-based distributed database like Mysql
 */

package io.jsql.sql.handler.data_mannipulation

import io.jsql.sql.OConnection

/**
 * Created by 长宏 on 2017/3/18 0018.
 * HANDLER tbl_name READ `PRIMARY` ...
 */
object Mhandler {
    fun isme(sql: String): Boolean {
        val sqll = sql.toUpperCase().trim { it <= ' ' }
        val list = sqll.split("\\s+".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        return list.size > 0 && list[0] == "HANDLER"
    }

    fun handle(sql: String, c: OConnection) {
        c.writeok()
    }
}
